cmake_modules/plugins.cmake

```diff
@@ -95,10 +95,7 @@ if (USE_AZURE)
     set(VCPKG_AZURE "${VCPKG_INCLUDE}")
 endif()
 
-set(VCPKG_BOOST_REGEX "${VCPKG_SUPPRESS}")
-if (USE_BOOST_REGEX)
-    set(VCPKG_BOOST_REGEX "${VCPKG_INCLUDE}")
-endif()
+set(VCPKG_BOOST_REGEX "${VCPKG_INCLUDE}")
```

Comment:
VCPKG_BOOST_REGEX is only needed when conditional, you can remove all instances of VCPKG_BOOST_REGEX here and then tweak `vcpkg.json.in` so its always included.

Reply:
Fixed.